### ISSUE TITLE
[BUG] Hoppers not filling stashes past increments of 64

### ISSUE THREAD

**Issue by Tomato-No-Mae** [Rebirth-of-the-Night/Rebirth-Of-The-Night#789]
**Describe the bug**
a hopper will send items to a stash until the stash reaches some multiple of 64 of an item (64,128, 192, etc)

**To Reproduce**  
place a hopper directed toward a stash, with more than 64 of an item

**Expected behavior**
the hopper should empty all of the items provided there is still room in the stash

**Screenshots**
![image](https://user-images.githubusercontent.com/67121000/127726236-f336ccfd-21e2-46c2-b04f-4639574ba943.png)
![image](https://user-images.githubusercontent.com/67121000/127726240-1bdb7f58-1b88-492d-9893-848b027f0fcf.png)

**Info (please complete the following information):**

 - Modpack Version:
   - [ ] 2.77.5
   - [x] 3.0 BETA 5
   - [ ] Other: 
 - Server or Singleplayer:
   - [ ] Server
   - [x] Singleplayer
 - Version the world was created on:
   - [ ] 2.77.5
   - [x] 3.0 BETA 5
   - [ ] Other: 

**Additional context**

**Comment by codetaylor** [comment]
Experimental fix in Pyrotech's next feature release `1.12.2-1.6.0`

**Comment by KaleThyself** [comment]
This issue has ***NOT** been resolved as of v.3.1.1a as item stack only goes up to 256 via hopper.

**Comment by democat3457** [comment]
correct, it will be resolved when we update to pyrotech 1.6.0, which will require a thorough evaluation when incorporating
